wasm/kws/assets/README.md

```diff
@@ -0,0 +1,27 @@
+# Introduction
+
+Please refer to
+https://www.modelscope.cn/models/pkufool/sherpa-onnx-kws-zipformer-wenetspeech-3.3M-2024-01-01/summary
+to download a model.
+
+# Kws
+
+The following is an example:
+```
+cd sherpa-onnx/wasm/kws
+git clone https://www.modelscope.cn/pkufool/sherpa-onnx-kws-zipformer-wenetspeech-3.3M-2024-01-01.git assets
+```
+
+You should have the following files in `assets` before you can run
+`build-wasm-simd-kws.sh`
```

Comment:
Please also check in this file.

You need to use
```bash
cd sherpa-onnx
git add -f ./build-wasm-simd-kws.sh
```

Reply:
👌